src/components/navBar/NavBar.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom';
+
+import './navBar.css';
+
+const NavBar = () => (
+  <div className="navContainer">
+    <h2>Bookstore CMS</h2>
+    <ul className="nav">
+      <li>
+        <Link to="/" className="link">BOOKS</Link>
+      </li>
+      <li>
+        <Link to="/categories" className="link">CATEGORIES</Link>
+      </li>
```

Comment:
Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 🤓 Well done!